sdk/communication/communication-call-automation/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "js",
   "TagPrefix": "js/communication/communication-call-automation",
-  "Tag": "js/communication/communication-call-automation_2662070cc1"
+  "Tag": "js/communication/communication-call-automation_545b84468a"
```

Comment:
Any reason why communication-call-automation is updated?

Reply:
There are some recordings that call the phone numbers api so I needed to update the api-version for those